tools/modsec2lua-resty-waf.pl

```diff
@@ -672,7 +672,7 @@ sub translate_options {
 			my $element = join '.', @elements;
 
 			# no $val, perhaps a delete?
-			if (!$val) {
+			if (!defined($val)) {
```

Comment:
one small style note, `defined` is a keyword, not a function, so no parenthesis around its argument